### PR TITLE
Better Settlers

### PR DESCRIPTION
**Pull request by ghostsquad**
After reading the [mod description](http://www.nexusmods.com/fallout4/mods/4772), I decided that it would be really useful to have this in loot if it wasn't already. I tested this locally.

### CODE REVIEW

**Review by Ortham — CHANGES_REQUESTED** [review]
If you get rid of the `# Better Settlers` and `# End Better Settlers` lines, I'll merge this. The comments aren't really useful and risk becoming outdated.

File: masterlist.yaml [hunk 1]
```diff
@@ -1399,3 +1399,32 @@ plugins:
       - 'WET.esp'
       - 'WET Murkier.esp'
       - 'WET Clearer.esp'
+      
+  # Better Settlers
+
```
Comment (Freso) [inline masterlist.yaml]: As @WrinklyNinja said, please remove these three lines…

File: masterlist.yaml [hunk 2]
```diff
@@ -1399,3 +1399,32 @@ plugins:
...
+      - 'BetterSettlersCleanFacePack.esp'
+      - 'BetterSettlersCCAPack1.1b.esp'
+      - 'BetterSettlersAllFemale.esp'
+      - 'BetterSettlersMortalPack.esp'
+      - 'BetterSettlersMortalSoldiersPack.esp'
+      - 'BetterSettlersLightArmorOnly.esp'
+      - 'BetterSettlersHeavyArmorOnly.esp'
+    inc: [ 'BetterSettlersFarHarborPatchMortal.esp' ]
+
+    # End Better Settlers
```
Comment (Freso) [inline masterlist.yaml]: And these two lines.

**Review by Freso — CHANGES_REQUESTED** [review]
Looks good apart from the few unneeded lines. 👍

**Comment by ghostsquad** [comment]
I've made the changes requested, please review.

**Comment by Ortham** [comment]
LGTM, thanks.

**Pull request merged**